package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "braintree-angular",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "Use Braintree in your Angular app",
-  "main": "index.js",
+  "main": "dist/braintree-angular.js",
```

Comment:
Is there a reason to change this in `package.json` as well? It might break if used as a dependency in browserify.